extensions/panache/rest-data-panache/deployment/src/main/java/io/quarkus/rest/data/panache/deployment/JaxRsResourceImplementor.java

```diff
@@ -107,6 +108,9 @@ private void implementClassAnnotations(ClassCreator classCreator, ResourceMetada
                 classCreator.addAnnotation(classAnnotation);
             }
         }
+        if (capabilities.isPresent(Capability.HIBERNATE_REACTIVE)) {
```

Comment:
I don't know how Panache REST Data works internally, but why is this actually needed? Is this class reused by the reactive version as well?

Reply:
Exactly, this implementor is used also for HIBERNATE classic, so we need to check the capabilities.